src/components/UI/Card.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { VscEdit } from "react-icons/vsc";
+import { VscSave } from "react-icons/vsc";
+import { VscClose } from "react-icons/vsc";
+
+import "./Card.css";
+
+class Card extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: false,
+      clicked: false,
+      inputValue: "",
+      textareaValue: "",
+      input: "",
+      textarea: ""
+    };
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    this.setState({input: this.state.inputValue});
+    this.setState({textarea: this.state.textareaValue});
+  }
+
+  inputChangeHandler(event) {
+    this.setState({ inputValue: event.target.value });
+  }
+
+  textareaChangeHandler(event) {
+    this.setState({ textareaValue: event.target.value });
+  }
```

Comment:
What if the card will be extended to have dozen of fields? Then you'll have to create dozen of handlers if follow the same logic... Try make it more dynamic. pass the required prop name as input attribute, then read it from the event object and set as dynamic property. So you'll have all handlers combined into one.